userbenchmark/triton/run.py

```diff
@@ -162,8 +162,8 @@ def _run(args: argparse.Namespace, extra_args: List[str]) -> BenchmarkOperatorRe
             else:
                 print(metrics)
         if not hasattr(torch_version, "git_version") and args.log_scuba:
-            from userbenchmark.triton.fb import log_benchmark
-            log_benchmark(metrics)
+            from pytorch.benchmark.fb.run_utils import log_benchmark
```

Comment:
why is the import changing?

Reply:
Oh nvm I see

Reply:
Xu added another scuba logging util into this run_utils and im consolidating the files. see [D61229059](https://www.internalfb.com/diff/D61229059) internally